### PR TITLE
INSTUI-2771 - FormField themeable migration

### PR DESCRIPTION
**Pull request by ke1k0**
Closes: INSTUI-2771

### CODE REVIEW

File: packages/ui-docs-client/src/Select/index.js [hunk 1]
```diff
@@ -105,6 +105,7 @@ class Select extends Component {
         renderBeforeInput={renderBeforeInput}
         inputValue={inputValue}
         isShowingOptions={isShowing}
+        onInputChange={() => {}}
```
Comment (matyasf) [inline packages/ui-docs-client/src/Select/index.js:108]: So this is the fix for the error that is thrown now when we load the docs client?

**Comment by matyasf** [comment]
There is something wrong with the commit plugin, it does not include the module name in the commit message :/

File: packages/ui-docs-client/src/Select/index.js (hunk 1 above)
Comment (ke1k0) [reply packages/ui-docs-client/src/Select/index.js:108]: Well yes, it is a required prop if value is provided, was missing before too... Couldn't figure out why it didn't throw error before :/

**Comment by ke1k0** [comment]
> There is something wrong with the commit plugin, it does not include the module name in the commit message :/

Modified the message, thanks.

**Review by Brailor — APPROVED** [review]

File: packages/ui-form-field/src/FormFieldLabel/styles.css [hunk 2]
```diff
@@ -1,27 +0,0 @@
...
-label.root,
-.root {
-  all: initial;
-  display: block;
-}
-
-.root.legend {
-  display: table;
-  width: 100%;
-}
```
[the remark this answers is not in the document]
Comment (ke1k0) [reply packages/ui-form-field/src/FormFieldLabel/styles.css:13]: it was probably some unused, undeleted style, the component never received "legend" class:
```js
    const classes = {
      [styles.root]: true,	
      [styles['has-content']]: hasVisibleChildren(this.props.children)	
    }
```

**Review by matyasf — APPROVED** [review]

**Pull request merged**